README.md

```diff
@@ -6,79 +6,147 @@
+[![PyPI](https://img.shields.io/pypi/v/pdm.svg?logo=python)](https://pypi.python.org/pypi/kaiko-eva)
+[![CI](https://github.com/kaiko-ai/eva/workflows/CI/badge.svg)](https://github.com/kaiko-ai/eva/actions?query=workflow%3ACI)
+[![license](https://img.shields.io/badge/License-Apache%202.0-blue.svg?labelColor=gray)](https://github.com/kaiko-ai/eva#license)
 
 <p align="center">
   <a href="#installation">Installation</a> •
   <a href="#how-to-use">How To Use</a> •
-  <a href="#datasets">Datasets</a> •
-  <a href="#contributing">Contribute</a>
+  <a href="https://kaiko-ai.github.io/eva/">Documentation</a> •
+  <a href="https://kaiko-ai.github.io/eva/dev/datasets/">Datasets</a> •
+  <a href="#benchmarks">Benchmarks</a> <br>
+  <a href="#contributing">Contribute</a> •
+  <a href="#acknowledgements">Acknowledgements</a>
 </p>
 
 </div>
 
----
+### Introduction
+
+`eva` is an evaluation framework for oncology foundation models (FMs) by [kaiko.ai](https://kaiko.ai/). Check out the [documentation](https://kaiko-ai.github.io/eva/) for more information.
+
+<div align="center">
 
-### _About_
+<img src="./docs/images/eva-process.gif" width="800">
 
-`eva` is [kaiko.ai](https://kaiko.ai/)'s evaluation framework for oncology foundation models (FMs). Check out the documentation (LINK TO BE ADDED) for more information.
+</div>
 
+### Highlights:
+- Easy and reliable benchmark of Oncology FMs
+- Automatic embedding inference and evaluation of a downstream task
+- Native support of popular medical [datasets](https://kaiko-ai.github.io/eva/dev/datasets/) and models
+- Produce statistics over multiple evaluation fits and multiple metrics
 
 ## Installation
 
-*Note: this section will be revised for the public package when publishing eva*
+Simple installation from PyPI:
+```sh
+# to install the core version only
+pip install kaiko-eva
 
-- Create and activate a virtual environment with Python 3.10+
+# to install the expanded `vision` version
+pip install 'kaiko-eva[vision]'
 
-- Install *eva* and the *eva-vision* package with:
+# to install everything
+pip install 'kaiko-eva[all]'
+```
 
+To install the latest version of the `main` branch:
+```sh
+pip install "kaiko-eva[vision] @ git+https://github.com/kaiko-ai/eva.git"
 ```
-pip install 'kaiko-eva[vision]'
+
+You can verify that the installation was successful by executing:
+```sh
+eva --version
 ```
 
-- To be able to use the existing configs, download them from the [*eva* GitHub repo](https://github.com/kaiko-ai/eva/tree/main) and move them to directory where you installed *eva*. 
+## How To Use
 
-### Run *eva*
+_eva_ can be used directly from the terminal as a CLI tool as follows:
+```sh
+eva {fit,predict,predict_fit} --config url/or/path/to/the/config.yaml 
+```
 
-Now you can run a complete *eva* workflow, for example with:
+For example, to perform a downstream evaluation of DINO ViT-S/16 on the BACH dataset with linear probing, by first infer the embeddings and perform 5 sequential fits, execute:
+```sh
```

Comment:
--> For example, to perform a downstream evaluation of DINO ViT-S/16 on the BACH dataset with linear probing by first inferring the embeddings and performing 5 sequential fits, execute:

Reply:
nice done